benchmark/benchmark_setup.hpp

```diff
@@ -28,7 +28,7 @@ namespace benchmark_setup
  */
 template<benchmark::TimeUnit time_unit>
 static void grid_sizes(benchmark::internal::Benchmark* bench) {
-    std::array<int, 5> sizes {256, 512, 1024, 2048, 4096};
+    std::array<int, 1> sizes {1024};
```

Comment:
I admit it's slow and not always very useful to run benchmarks for multiple grid sizes, although in some cases this could give a rough idea on how scales an algorithm applied on the grid.

So maybe let's define another Apply function (or let's just not use this one) for the benchmarks where we don't need to apply multiple grid sizes?

Reply:
Sure! I was just annoyed because large grid need to have a higher bench execution time to get more than a single run.
Will rollback this modification.

Do you think that we need to keep this for physical algorithms while benchmarks of the base elements could be done with 1 or 2 sizes?